invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/ShareOptions/AccessLinks/CreateAccessLink.js

```diff
@@ -21,6 +21,24 @@ export class CreateAccessLink extends Component {
     };
   }
 
+  dropdownOptionsGenerator = (value) => {
```

Comment:
could we move it to react-invenio-forms and create a reusable utility function? 
this small snippet is copied over and over otherwise